Modules/Challenges/test/Challenges.API.Tests.Performance.EndToEnd/src/post-with-authentication.test.ts

```diff
@@ -0,0 +1,63 @@
+import http from 'k6/http';
+import { Options } from 'k6/options';
+import { describe, expect } from "https://jslib.k6.io/k6chaijs/4.3.4.2/index.js";
+
+const host = __ENV.HOST;
+const apiEndpoint = host+'/api/v1/';
+
+function translateSize() {
+    switch(__ENV.SIZE){
+        case "S": return { vus: 1, iterations: 10 };
+        case "M": return { vus: 10, iterations: 50 };
+        case "L": return { vus: 50, iterations: 100 };
+        default: throw new Error("Invalid 'Size' value");
+    }
+}
+
+export const options: Options = {
```

Comment:
Don't export the options

Reply:
[https://k6.io/docs/using-k6/test-lifecycle/](url)
Considering the previous link I believe that options need to be exported, since they are processed in the Init stage of the lifecycles.